index.html

```diff
@@ -22,9 +22,10 @@
     <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
       <div class="navbar-nav">
         <a class="nav-item nav-link active" id="log-in-nav" href="#">Log In</a>
+        <a class="nav-item nav-link hidden" id="log-out-nav" href="#">Log Out</a>
         <a class="nav-item nav-link" id="sign-up-nav" href="#">Register</a>
         <!-- <a class="nav-item nav-link" href="#">Profile</a> -->
-        <a class="nav-item nav-link" id="glean-req-nav" href="#">Gleaning Request</a>
+        <a class="nav-item nav-link hidden" id="glean-req-nav" href="#">Gleaning Request</a>
```

Comment:
This link is only viewable if you logged in as an existing user. Not viewable after a new user registration

Reply:
Good catch. I believe it is working now, but please test again.